.size-snapshot.json

```diff
@@ -1,13 +1,13 @@
 {
   "bundle.js": {
-    "bundled": 997397,
-    "minified": 929528,
-    "gzipped": 99130
+    "bundled": 900134,
```

Comment:
Removing a bunch of `css` usage continues to reduce the bundle size considerably. 

Since it isn't triggering regression in Chromatic, my expectation is that this'll be fine in consumer component usage, but just calling out that it's possible we might have to add more of the calls back in order to get things working, which will increase the bundle again.